README.md

```diff
@@ -71,20 +71,22 @@ var client = new IdxClient(new IdxConfiguration()
                 RedirectUri = "{YOUR_REDIRECT_URI}", // Must match the redirect uri in client app settings/console
             });
 ```
-#### Provide your device context to be used during the client's bootstrapping
+#### Provide your request context to be used during the authentication flow bootstrapping
 
-The device context object specifies the headers to be used during the client's bootstrapping. These headers are *only* sent when calling the /interact endpoint.
+The request context object specifies the headers to be used in methods that bootstraps a new authentication flow. In other words, these headers are *only* sent when calling the /interact endpoint.
 
 ```csharp
-var deviceContext = new DeviceContext();
-deviceContext.UseHeader(RequestHeaders.XDeviceToken, "deviceToken"); // Only for confidential clients
-deviceContext.UseHeader(RequestHeaders.UserAgent, "MyUserAgent");
-deviceContext.UseHeader(RequestHeaders.XOktaUserAgentExtended, "MyXUserAgent");
-deviceContext.UseHeader(RequestHeaders.XForwardedFor, "MyForwardedFor");
+var requestContext = new RequestContext
+                    {
+                        XDeviceToken = "deviceToken",
+                        XOktaUserAgentExtended = "MyXUserAgent",
+                        XForwardedFor = "MyForwardedFor",
+                    };
```

Comment:
Suggestion (not a blocker)

From a developer's perspective, these names leak the actual implementation.  For example, this constructor needs the User-Agent string, but the developer likely won't know that the user-agent actually maps back to `X-Okta-User-Agent-Extended` header, similar to `X-Forwarded-For`, which is an IP address.

The actual implementation could change to move these into the POST body with different attribute names.

Consider changing the names to, `DeviceToken`, `UserAgent`, and `IpAddress`:

```csharp
var requestContext = new RequestContext
    {
        DeviceToken = "deviceToken",
        UserAgent = "MyUserAgent",
        IpAddress = "MyIpAddress",
    };
```

Reply:
Thanks for the suggestion Demers! I removed the X prefix from DT and UA but we're keeping the XFF name to be consistent with classic.